Themes/default/Register.template.php

```diff
@@ -300,7 +313,7 @@ function verifyAgree()
 				<input type="submit" name="accept_agreement_coppa" value="', $context['coppa_agree_below'], '" class="button">';
 	else
 		echo '
-				<input type="submit" name="regSubmit" value="', $txt['register'], '" tabindex="', $context['tabindex']++, '" class="button">';
+				<input type="submit" name="regSubmit" value="', $txt['agreement' . (!empty($context['require_policy_agreement']) ? '_policy' : '') . '_agree'], '" class="button" />';
```

Comment:
I don't think you want to edit this button, this is the `Signup` submit button, instead we want to edit the buttons with the name `accept_agreement` in this same template